compiler/ir/ir.tree/src/org/jetbrains/kotlin/ir/util/RenderIrElement.kt

```diff
@@ -510,7 +510,7 @@ class RenderIrElementVisitor(private val normalizeNames: Boolean = false) : IrEl
         renderFlagsList(
             "vararg".takeIf { varargElementType != null },
             "crossinline".takeIf { isCrossinline },
-            "noinline".takeIf { isNoinline }
+            "noinline".takeIf { isNoinline },
```

Comment:
Seems like it would be good adding `isAssignagle` into render too

Reply:
Done.